upi/openstack/inventory.yaml

```diff
@@ -26,7 +26,7 @@ all:
       # OpenShift cluster resources. It is built in the form of:
       #
       #     "{{ os_cluster_name }}{{ installation-specific string }}"
-      os_infra_id: "{{ os_cluster_name }}-upi"
+      os_infra_id: "{{ os_cluster_name }}"
```

Comment:
oh yes, thank you for this. That `upi` was meant to be a placeholder for the random string generated by the installer; however, it makes little sense to change a value the user provided explicitly. :+1:

Reply:
I think `os_infra_id` must be unique and we shouldn't allow users to specify it at all. If there are two clusters with the same infra ID, then there may be resource collisions. Also as you remember the installer destroys clusters based on their os_infra_id values, so we can accidentally destroy another cluster with the same infra ID.